src/web/mjs/connectors/BentoManga.mjs

```diff
@@ -0,0 +1,79 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class bentomanga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'bentomanga';
+        super.label = 'Bentô Manga';
+        this.tags = [ 'manga', 'webtoon', 'novel', 'french' ];
+        this.url = 'https://bentomanga.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'h1'); 
+        let id = uri.pathname + uri.search;
+        let title = data[0].textContent.trim(); 
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(this.url + '/manga_list', this.requestOptions); 
```

Comment:
This method is fine by itself but Bentomanga got more than 1k manga pages. 
Suggestion to use _getmangas and _getMangasFromPage from [Sample.mjs](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/templates/%40Samples.mjs)

If you want to be efficient, you can get JSON manga with html data inside by calling 
https://bentomanga.com/manga_list?limit=X&cb=Y . 

X being page number-1, cb being the timestamp from when you call the page.
This request must get a special header, with name "X-Requested-With" and value "XMLHttpRequest". 

Then you get some html in the json ("mangas")

```
	<div class="div-manga_cover">
		<a href="https://bentomanga.com/manga/maseki-gurume-mamono-no-chikara-o-tabeta-ore-wa-saikyou" class="component-manga-cover">
			<img alt="couverture manga" src="https://bentomanga.com/media/cache/manga_medium/images/mangas/cover/5e47469782828797082475.jpg">
				<span class="div-manga_cover-title">Maseki Gurume: Mamono no Chikara o Tabeta Ore wa Saikyou!</span>
		</a>
		<span class="badge-chapterCount mobile-adapt">0/29</span>
	</div>
```

There are plenty of sample about how to do that in other connectors.


        request.headers.set('X-Requested-With', 'XMLHttpRequest'); 
        const data = await this.fetchJSON(request);
        const dom =  this.createDOM( data.mangas ); //json got html  data in "mangas"
        const mangaNode = dom.querySelector('div.div-manga_cover a');
        return mangaNode.map(element => {
            return {
                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
                title: element.querySelector('span.div-manga_cover-title').textContent.trim()
            };
        });